packages/myst-cli-utils/src/utils.ts

```diff
@@ -60,5 +62,8 @@ export function plural(f: string, count?: number | any[] | Record<any, any>): st
       : Array.isArray(count)
       ? count?.length
       : Object.keys(count ?? {}).length) ?? 0;
-  return f.replace('%s', String(num)).replace(/\(s\)/g, num === 1 ? '' : 's');
+  return f
+    .replace('%s', String(num))
+    .replace(/\(s\)/g, num === 1 ? '' : 's')
+    .replace(/\(([a-z0-9A-Z-]*)\|([a-z0-9A-Z-]*)\)/g, num === 1 ? '$1' : '$2');
```

Comment:
You can replace line 67 and 68 with this:

```suggestion
    .replace(/\((([a-z0-9A-Z-]*)\|)?([a-z0-9A-Z-]*)\)/g, num === 1 ? '$2' : '$3');
```

In addition to only being one regex, it handles additional cases that were not covered before, e.g. `stitch(es)`.

Reply:
(Would be good to chuck some tests on this though...)

Reply:
Time for some tests, yeah. :)

Reply:
Brought in your changes and added tests!